mass_mailing_partner/models/mail_mass_mailing_contact.py

```diff
@@ -40,30 +40,38 @@ def _onchange_partner_mass_mailing_partner(self):
 
     @api.model
     def create(self, vals):
-        record = self.new(vals)
+        # Ensure that defaults are loaded (e.g.: import csv or xls)
+        values_w_defaults = self.default_get(self._fields.keys())
```

Comment:
I'm not saying this is wrong, but it makes me wonder... why aren't the default values used by default? :thinking: 

I thought that's what "defaults" are for.

Reply:
What do you think about the first approach?: https://github.com/OCA/social/commit/4f705c504b3cb855db6aafebb528dd6610c91000

Reply:
The problem is that a new record doesn't apply defaults, and when converting cache to values you lose that default, as the returned dictionary contains all the fields without values, so there's no opportunity of defaults to be applied as "specifically" a value is provided. Do you get it? I still prefer this approach in create than the previous one, as it's specific for that thing, and let out any other possible default.

Reply:
Ok, now I understand also :) Let's go with it